src/model/rpc.js

```diff
@@ -44,9 +44,17 @@ module.exports = class HttpRpc extends EventEmitter {
       })
     ])
 
+    // 401 is a common error when a peering relationship isn't mutual, so a more
+    // helpful error is printed.
+    if (result.statusCode === 401) {
+      throw new Error('Unable to call "' + prefix +
```

Comment:
instead of `prefix` (which will look like "peer.a1Mg_.usd.9"), report `this.rpcUri` (which will look like "https://pineapplesheep.ilp.rocks/api/rpc")